mmdet3d/models/backbones/pointnet2_sa_msg.py

```diff
@@ -37,27 +40,42 @@ class PointNet2SAMSG(BasePointNet):
               each SA module.
     """
 
-    def __init__(self,
-                 in_channels,
-                 num_points=(2048, 1024, 512, 256),
-                 radii=((0.2, 0.4, 0.8), (0.4, 0.8, 1.6), (1.6, 3.2, 4.8)),
-                 num_samples=((32, 32, 64), (32, 32, 64), (32, 32, 32)),
-                 sa_channels=(((16, 16, 32), (16, 16, 32), (32, 32, 64)),
-                              ((64, 64, 128), (64, 64, 128), (64, 96, 128)),
-                              ((128, 128, 256), (128, 192, 256), (128, 256,
-                                                                  256))),
-                 aggregation_channels=(64, 128, 256),
-                 fps_mods=(('D-FPS'), ('FS'), ('F-FPS', 'D-FPS')),
-                 fps_sample_range_lists=((-1), (-1), (512, -1)),
-                 dilated_group=(True, True, True),
-                 out_indices=(2, ),
-                 norm_cfg=dict(type='BN2d'),
-                 sa_cfg=dict(
-                     type='PointSAModuleMSG',
-                     pool_mod='max',
-                     use_xyz=True,
-                     normalize_xyz=False),
-                 init_cfg=None):
+    def __init__(
+            self,
+            in_channels: int,
+            num_points: Sequence[int] = (2048, 1024, 512, 256),
+            radii: Sequence[Tuple[float, float, float]] = (
+                (0.2, 0.4, 0.8),
+                (0.4, 0.8, 1.6),
+                (1.6, 3.2, 4.8),
+            ),
+            num_samples: Sequence[Tuple[int, int,
```

Comment:
This type-hint is too long and ugly. Please modify these like: https://github.com/open-mmlab/mmdetection3d/blob/dev-1.x/mmdet3d/evaluation/functional/panoptic_seg_eval.py#L7